templates/db.cfn.yml

```diff
@@ -1,6 +1,8 @@
 ---
 AWSTemplateFormatVersion: 2010-09-09
```

Comment:
Nit, but consider changing the name of this file to rds-db.cfn.yml?

Reply:
I'm totally conflicted on this one. Aurora is RDS too. I am not going to address in this PR, but I will look at in the future.  SIM added for bastion change and assigned to @koroshan